src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs

```diff
@@ -377,6 +377,12 @@ public async Task<IActionResult> MoveMediaList(string[] mediaNames, string sourc
                 path = "";
             }
 
+            var invalidFolderNameCharacters = new char[] { '\\', '/' };
+            if (invalidFolderNameCharacters.Any(invalidChar => name.Contains(invalidChar)))
+            {
+                return StatusCode(StatusCodes.Status403Forbidden, S["Cannot create folder because the folder name contains invalid characters"]);
```

Comment:
Looks great!

Call me Mr Picky but isn't model state errors `BadRequest` ?

https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.statuscodes.status400badrequest?view=aspnetcore-5.0

Reply:
@deanmarcussen Yes, it is, I changed it.